checker/tests/interning/Raw3.java

```diff
@@ -54,7 +54,7 @@
   // with raw types
   List<@Interned String> foo6() {
     List<String> sl = new ArrayList<>();
-    //:: warning: [unchecked] unchecked conversion
+    //:: warning: [unchecked] unchecked conversion :: error: (return.type.incompatible)
```

Comment:
Also, in this particular case, the type arguments for the raw type are correctly inferred.